processing/src/main/java/org/apache/druid/query/filter/InDimFilter.java

```diff
@@ -132,31 +145,40 @@ public FilterTuning getFilterTuning()
   @Override
   public byte[] getCacheKey()
   {
```

Comment:
Can multiple threads call `getCacheKey` at the same time?

What happens if 2 threads try to build sortedValues?

Reply:
All `DimFilter`s are not thread-safe. It should be never called by two threads at the same time.